lib/mempool/mempool.js

```diff
@@ -193,13 +202,6 @@ class Mempool extends EventEmitter {
    */
 
   async _addBlock(block, txs, view) {
-    if (this.map.size === 0
```

Comment:
Why was this removed? (same as in another PR)

Reply:
https://github.com/handshake-org/hsd/pull/773#discussion_r1062779385